qa-tests-backend/src/main/groovy/util/SplunkUtil.groovy

```diff
@@ -141,6 +133,26 @@ class SplunkUtil {
         }
     }
 
+    static String createHECToken(int port, String tokenName = "stackrox") {
+        Response response = null
+        withRetry(20, 15) {
+            response =  given().auth()
+                .basic("admin", SPLUNK_ADMIN_PASSWORD)
+                .formParam("name", tokenName)
+                .formParam("output_mode", "json")
+                .post("https://127.0.0.1:${port}/servicesNS/nobody/search/data/inputs/http")
+        }
+        return unmarshalHEC(response?.asString())
```

Comment:
It would be good to assert here if `unmarshalHEC` returned a valid token. Maybe this becomes redundant if we already asserted that the response is valid earlier.

Reply:
I assert on the return of `unmarshalHEC`, as this is the function that retrieves the raw data